### PR TITLE
feat: Added a Script which changes Desktop Wallpaper (Windows). Issue…

### PR DESCRIPTION
**Pull request by symtalha14**
… #349

Added a new folder Windows_Wallpaper_Changer in Scripts/Miscellaneous. It has got a simple python script which changes the windows desktop wallpaper by prompting the user to select an image.

# Description

A short summary of what is included in your Pull Request.

Fixes #(issue_no)

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [ ] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [ ] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Miscellaneous/Windows_Wallpaper_Changer/background.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from tkinter import Tk     # from Tkinter import Tk for Python 2.x
+from tkinter.filedialog import askopenfilename
+import ctypes
+import os
+
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/Windows_Wallpaper_Changer/background.py:5]: This import is not used anywhere, please remove this!

**Review by tusharnankani — CHANGES_REQUESTED** [review]
The script works fine! @symtalha14 👍🏻 

Please add a README.md file in your directory according to [this template](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)!

**Review by tusharnankani — APPROVED** [review]

File: Scripts/Miscellaneous/Windows_Wallpaper_Changer/background.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from tkinter import Tk     # from Tkinter import Tk for Python 2.x
+from tkinter.filedialog import askopenfilename
+import ctypes
+
+
+
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/Windows_Wallpaper_Changer/background.py:6]: Remove the extra lines.

**Comment by symtalha14** [comment]
I hope everything is in place now :)

**Comment by symtalha14** [comment]
Could you please consider merging if everything is alright :')

**Comment by tusharnankani** [comment]
It will automatically get merged, when two or more maintainers/owners approve the changes! 

Thanks to our very own - `mergify` bot. ^_^

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @symtalha14 for code

**Pull request merged**